README.md

```diff
@@ -1104,6 +1105,48 @@ bash -v scriptname
 bash -x scriptname
 ```
 
+# 5. Package Management
+There are two types of packages available for linux systems.
+* Debian Packages (.deb)
+* RedHat Package Manager (.rpm)
+
+## 5.1 Debian Package (deb)
+deb is the format, as well as extension of the software package format for the Debian distribution and its derivatives.
+
+### 5.1 A.`Update Packages`
+The Debian package manager updates its package data by ```bash sudo apt update```
```

Comment:
You should remove `bash` (because it's a bash guide we suppose the user is already running bash) and maybe `sudo` which isn't needed in root and not mandatory for some apt commands